components/02-content/conclusion/conclusion.njk

```diff
@@ -0,0 +1,25 @@
+<ul class="dso-conclusion">
+  {%- for item in items -%}
+    <li class="dso-{{ item.modifier }}">
+      {{ item.label | safe }}
+      {%- if item.info %}
+        {% render '@info-button', item.info %}
+      {% endif %}
+
+      {%- if item.subItems %}
+        <ul>
+          {%- for subItem in item.subItems -%}
+            <li>{{ subItem | safe }}</li>
+          {% endfor -%}
+        </ul>
+      {% endif -%}
+
+      {%- if item.info %}
+        {% render '@info', item.info %}
+      {% endif -%}
+    </li>
+  {% endfor -%}
+</ul>
+{% if alert %}
+  {% render '@alert', alert %}
```

Comment:
In handlebars stond er `{{render '@alert' alert merge=true }}`, moet er in Nunjucks niet `{% render '@alert', alert, true %}` staan?

Reply:
Idem voor de `render`s op regel 6 en regel 18?

Reply:
Ik zag geen verschillen in de render files, maar heb ze toch inderdaad maar toegevoegd. Het stond er waarschijnlijk niet zonder reden.